v2_1/ws/rest/docs/4_1_introduction.md

```diff
@@ -1,19 +1,20 @@
 # SDMX RESTful API
 
-## A Brief Introduction to REST
+This SDMX API is based on the REST principles
 
-This SDMX API is based on the REST principles, as described below:
+- In REST, specific information is known as **Resource**. In SDMX, specific resources would be, for example, code lists, concept schemes, data structure definitions, dataflows, etc. Each resource is addressable via a **global identifier** (i.e.: a URI).
```

Comment:
Suggestion: ... In SDMX, specific resources would be **_maintainable artefacts_**, for example, ...

Reply:
Done, thanks @dosse